src/components/message-box/MessageBox.tsx

```diff
@@ -39,53 +59,78 @@ export const MessageBox = ({
   const colorStyle = {
     backgroundColor: theme.static.status[type].background,
   };
+
+  const onPress =
+    onPressConfig &&
+    ('action' in onPressConfig
+      ? onPressConfig.action
+      : () => Linking.openURL(onPressConfig.url));
+
+  const a11yLabel = [title, message, onPressConfig?.text]
+    .filter((s): s is string => !!s)
```

Comment:
There are other cases for this kind of filter where `.filter(Bool)` is used instead.

Reply:
When using `.filter(Boolean)` it doesn't work as a [type predicate](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates) since we are missing the `s is string` part. This would make `a11yLabel` a `(string | undefined)[]` and not `string[]`.